server.ts

```diff
@@ -191,4 +72,4 @@ if (moduleFilename === __filename || moduleFilename.includes('iisnode')) {
   run();
 }
 
-export * from './src/main.server';
+export default bootstrap;
```

Comment:
I'm not sure exactly what all the changes in this file do - they were generated automatically by me running the the `ng update` command with arguments as shown in git history.